src/nile/common.py

```diff
@@ -28,6 +28,10 @@
     # subject to change
     "0x041a78e741e5af2fec34b695679bc6891742439f7afb8484ecd7766661ad02bf"
 )
+OTHER_NETWORKS = {
```

Comment:
```suggestion
DEFAULT_GATEWAYS = {
```

I think this is more consistent.